src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -77,6 +77,11 @@ export const AdminAdvancedSettings: React.FC = () => {
     return <LoadingPage />;
   }
 
+  if (!venue) {
+    //@debt Add NotFound page here after it's merged
```

Comment:
Why after merge?

Reply:
What I mean by that is after the PR for NotFound component is merged then we need to add it here. Or just in that PR. https://github.com/sparkletown/sparkle/pull/2219